queue/src/fiskaltrust.Middleware.Localization.QueueIT/RequestCommands/InitialOperationReceiptCommand.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Threading.Tasks;
+using fiskaltrust.storage.V0;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.storage.serialization.DE.V0;
+using fiskaltrust.Middleware.Localization.QueueIT.Factories;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace fiskaltrust.Middleware.Localization.QueueIT.RequestCommands
+{
+    public class InitialOperationReceiptCommand : Contracts.RequestCommands.InitialOperationReceiptCommand
+    {
+        public override long CountryBaseState => Constants.Cases.BASE_STATE;
+
+        private readonly IConfigurationRepository _configurationRepository;
+        private readonly SignatureItemFactoryIT _signatureItemFactoryIT;
+        private readonly ftQueueIT _queueIt;
+
+        public InitialOperationReceiptCommand(ILogger<InitialOperationReceiptCommand> logger, IConfigurationRepository configurationRepository, SignatureItemFactoryIT signatureItemFactoryIT, ftQueueIT queueIt) : base(logger)
+        {
+            _configurationRepository = configurationRepository;
+            _signatureItemFactoryIT = signatureItemFactoryIT;
+            _queueIt = queueIt;
+        }
+
+        protected override async Task<(ftActionJournal, SignaturItem)> InitializeSCUAsync(ftQueue queue, ReceiptRequest request, ftQueueItem queueItem)
```

Comment:
We also need to call the base method here, right?

Reply:
the base is calling this